packages/main/src/AvatarGroup.ts

```diff
@@ -233,25 +138,120 @@ const metadata = {
+* @param {boolean} overflowButtonClicked indicates if the overflow button is clicked
+* @event sap.ui.webc.main.AvatarGroup#click
+* @public
+* @since 1.0.0-rc.11
+*/
+@event("click", {
+	detail: {
+		targetRef: { type: HTMLElement },
+		overflowButtonClicked: { type: Boolean },
+	},
+})
+
+/**
+* Fired when the count of visible <code>ui5-avatar</code> elements in the
+* component has changed
+* @event sap.ui.webc.main.AvatarGroup#overflow
+* @public
+* @since 1.0.0-rc.13
+*/
+@event("overflow")
+
 class AvatarGroup extends UI5Element {
+	/**
+	 * Defines the mode of the <code>AvatarGroup</code>.
+	 * <br><br>
+	 * Available options are:
+	 * <ul>
+	 * <li><code>Group</code></li>
+	 * <li><code>Individual</code></li>
+	 * </ul>
+	 * @type {sap.ui.webc.main.types.AvatarGroupType}
+	 * @name sap.ui.webc.main.AvatarGroup.prototype.type
+	 * @defaultValue "Group"
+	 * @public
+	 */
+	@property({ type: AvatarGroupType, defaultValue: AvatarGroupType.Group })
+	type!: AvatarGroupType
+
+	/**
+	 * Defines the aria-haspopup value of the component on:
+	 * <br><br>
+	 * <ul>
+	 * <li> the whole container when <code>type</code> property is <code>Group</code></li>
+	 * <li> the default "More" overflow button when <code>type</code> is <code>Individual</code></li>
+	 * </ul>
+	 * <br><br>
+	 * @type String
+	 * @name sap.ui.webc.main.AvatarGroup.prototype.ariaHaspopup
+	 * @since 1.0.0-rc.15
+	 * @protected
+	 */
+	@property()
+	ariaHaspopup!: string
+
+	/**
+	 * @private
+	 */
+	@property({ noAttribute: true })
+	_overflowButtonText!: string
+
+	/**
+	 * Defines the items of the component. Use the <code>ui5-avatar</code> component as an item.
+	 * <br><br>
+	 * <b>Note:</b> The UX guidelines recommends using avatars with "Circle" shape.
+	 * Moreover, if you use avatars with "Square" shape, there will be visual inconsistency
+	 * as the built-in overflow action has "Circle" shape.
+	 * @type {sap.ui.webc.main.IAvatar[]}
+	 * @name sap.ui.webc.main.AvatarGroup.prototype.default
+	 * @slot items
+	 * @public
+	 */
+	@slot({ type: HTMLElement, "default": true })
+	items!: Array<Avatar>;
+
+	/**
+	 * Defines the overflow button of the component.
+	 * <b>Note:</b> We recommend using the <code>ui5-button</code> component.
+	 * <br><br>
+	 * <b>Note:</b> If this slot is not used, the component will
+	 * display the built-in overflow button.
+	 * @type {HTMLElement}
+	 * @name sap.ui.webc.main.AvatarGroup.prototype.overflowButton
+	 * @slot overflowButton
+	 * @public
+	 * @since 1.0.0-rc.13
+	 */
+	@slot()
+	overflowButton!: Array<Button>;
+
+	static i18nBundle: I18nBundle;
+	_onResizeHandler: () => void;
+	// _itemNavigation: ItemNavigation;
+	_colorIndex: number;
+	_hiddenItems: number;
+	_itemNavigation: ItemNavigation;
+
 	constructor() {
 		super();
 
 		this._itemNavigation = new ItemNavigation(this, {
-			getItemsCallback: () => {
-				return this._isGroup ? [] : this.items.slice(0, this._hiddenStartIndex);
-			},
+			getItemsCallback: () => this.items,
```

Comment:
`getItemsCallback` used to return the items filtered based on whether isGroup or there are hidden items

deleted:
```js 
return this._isGroup ? [] : this.items.slice(0, this._hiddenStartIndex);